[Build] mp11 not found

### Describe the issue

Building the latest version of the main branch results in a succes, however when then running `sudo make install` the install section fails with the output below.

The same happens when not building with cuda or tensorrt.

What could be the problem here?

### Urgency

_No response_

### Target platform

Jetson Orin AGX

### Build script

 ./build.sh --config Release --update --build --parallel 10 --nvcc_threads 4 --build_shared_lib --use_cuda --cuda_home /usr/local/cuda --cudnn_home /usr/lib/$(uname -m)-linux-gnu --use_tensorrt --tensorrt_home /usr/lib/$(uname -m)-linux-gnu --allow_running_as_root --skip_tests --skip_submodule_sync --cmake_extra_defines CMAKE_CUDA_ARCHITECTURES="87" CMAKE_CXX_FLAGS="-Wno-unused-variable" onnxruntime_BUILD_UNIT_TESTS=OFF

### Error / output

[ 27%] Built target absl_log_internal_check_op
[ 27%] Built target absl_log_internal_conditions
[ 28%] Built target onnxruntime_common
[ 34%] Built target onnxruntime_mlas
[ 35%] Built target cpuinfo
[ 35%] Built target onnxruntime_mlas_q4dq
[ 36%] Built target onnxruntime_graph
[ 41%] Built target onnxruntime_framework
[ 41%] Built target onnxruntime_lora
[ 42%] Built target onnxruntime_util
[ 45%] Built target onnx
[ 59%] Built target onnxruntime_providers
[ 59%] Built target onnxruntime_providers_shared
[ 92%] Built target onnxruntime_providers_cuda
[ 93%] Built target onnxruntime_providers_tensorrt
[ 98%] Built target onnxruntime_optimizer
[ 98%] Built target onnxruntime_session
[ 98%] Built target onnxruntime_generate_def
[100%] Built target re2
[100%] Built target onnxruntime
Install the project...
-- Install configuration: "Release"
CMake Error at cmake_install.cmake:47 (include):
  include could not find requested file:

    /opt/onnxruntime/build/Linux/Release/_deps/mp11-build/cmake_install.cmake

### Visual Studio Version

_No response_

### GCC / Compiler Version

_No response_

--- a/cmake/external/onnxruntime_external_deps.cmake
+++ b/cmake/external/onnxruntime_external_deps.cmake
@@ -301,6 +301,7 @@ if(NOT TARGET Boost::mp11)
     onnxruntime_fetchcontent_declare(
      mp11
      URL ${DEP_URL_mp11}
+     EXCLUDE_FROM_ALL
      FIND_PACKAGE_ARGS NAMES Boost
     )
     onnxruntime_fetchcontent_makeavailable(mp11)    

